app/Http/routes.php

```diff
@@ -65,4 +65,11 @@
   Route::get('/jobs/owned', 'JobsController@owned');
 });
 
+Route::get('/blog/{page}', function($page){
```

Comment:
Any reason this isn't in a controller class. I know it's not much code but I still think it warrants it's own Controller.